_schemas/1.0/schema_create_resource.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
regarding the schema version we will be using in v1.0, we need to get input from community experts. may be using the latest schema version with v1.0 specs is OK somehow

Reply:
I don't see any harm in using the latest version, but it's up to you. Let's see what the community experts think about it.

Reply:
By now i would say, just use the latest version